tests/Dfc.CourseDirectory.Testing/DataStore/CosmosDb/QueryHandlers/GetProviderUkprnForCourseHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+
+namespace Dfc.CourseDirectory.Testing.DataStore.CosmosDb.QueryHandlers
+{
+    public class GetProviderUkprnForCourseHandler : ICosmosDbQueryHandler<GetProviderUkprnForCourse, int?>
```

Comment:
Not obvious that this is a mock/in memory implementation. Consider renaming or namespacing?

Reply:
All the implementations in the namespace are in-memory implementations. Happy to look at the naming but I'd rather do them all together in a different PR.